### DISCUSSION TITLE
Error

### DISCUSSION THREAD

**Discussion by Abdullahoda** [IBAX-io/go-ibax#67] [Bug Submission]
abdullah oda:
Price can be understood as the money or amount to be paid to get something. Cost is the amount incurred in the production of goods, i.e. the money value of the resources involved in producing something.

4 characters  the fee is

I think there is a typo here

If you want to award

Piroglu3841

Twitter 😁
![Uploading IMG_20220118_151404.jpg…]()

**Comment by HalloKyle** [comment]
Could you provide a full description for that place？

**Reply by Abdullahoda** [reply]
How can I explain more?

**Reply by Abdullahoda** [reply]
you see in the photo

**Reply by Abdullahoda** [reply]
![Screenshot_20220122_163615](https://user-images.githubusercontent.com/97207588/150640760-6cedcb20-40b8-428d-8bb3-7a8d4b4b0514.jpg)

**Reply by Despair016** [reply]
> Could you provide a full description for that place？

@BadAssKyle I also gave 2 suggestions Why don't you read and make positive or negative comments?

**Reply by Abdullahoda** [reply]
I could not get an answer

**Comment by Abdullahoda** [comment]
![Screenshot_20220122_163615](https://user-images.githubusercontent.com/97207588/150640626-469b0fd8-55b0-4aea-8c14-f6a899b6c83b.jpg)

**Reply by Abdullahoda** [reply]
Instead of the cost is it says the free is they should all be the same

**Comment by Abdullahoda** [comment]
I could not get an answer
